src/main/java/core/mate/academy/service/MachineServiceImpl.java

```diff
@@ -12,7 +12,7 @@
 /**
  * Your implementation of MachineService.
  */
-public class MachineServiceImpl<T> implements MachineService<T> {
+public class MachineServiceImpl<T extends Machine> implements MachineService<Machine> {
```

Comment:
```suggestion
public class MachineServiceImpl implements MachineService<Machine> {
```
як варіант можна навіть так